widgets/src/main/java/org/pentaho/gwt/widgets/client/utils/NameUtils.java

```diff
@@ -76,9 +76,13 @@ public static String reservedCharListForDisplay( ) {
     return getReservedCharsDisplay();
   }
 
-  public static native String URLEncode( String value )
+  public static String URLEncode( String value ) {
```

Comment:
Is the method name URLEncode (missing an 'r') trying to be consistent with class in other projects `impl/client/src/main/javascript/web/util/URLEncoder.js` ?